scripts/simplify_network.py

```diff
@@ -522,6 +523,64 @@ def consense(x):
     return clustering.network, clustering.busmap
 
 
+def merge_isolated_buses(n, drop_p_threshold=0):
+    generators_sum_origin = n.generators.p_nom.sum()
+    load_sum_origin = n.loads_t.p_set.sum().sum()
+
+    # duplicated sub-networks mean that there is at least one interconnection between buses
+    i_islands = n.buses[
+        (~n.buses.duplicated(subset=["sub_network"], keep=False))
+        & (n.buses.carrier == "AC")
```

Comment:
Do we need this limitation to AC? I'd remove this limitation unless necessary for some reasons

Reply:
That is a quick (and not the cleanest) fix for DC-links. If we are using unique sub-networks as the only criterium, such DC-link-buses are treated as islanded

Reply:
ok! let's keep them for now